golden/config.go

```diff
@@ -78,6 +78,8 @@ type BashConfig struct {
 	// WorkingDir is the directory where the bash script(s) will be
 	// executed.
 	WorkingDir string
+	// WaitBefore adds a delay before running the bash script to handle rate limits.
```

Comment:
```suggestion
	// WaitBefore adds a delay before running the bash script. This is useful
	// when throttling is needed, e.g., when dealing with rate limiting.
```

Just to suggest that there may be other uses for this in the future, as this is a shared package. :relaxed: